### DISCUSSION TITLE
Transparent input padding is not applied inside form

### DISCUSSION THREAD

**Discussion by ShiiRochi** [fomantic/Fomantic-UI#815] [Elements (button, image, label, loader, ...)] [answered]
# Bug Report

When using transparent input inside a form its padding is overridden by those, applied to input when it is inside form.

I will write in emmet abbrs, so that it would take less time to describe

## Steps to reproduce
1. create .ui.form container
2. place there .ui.transparent,input > input[type=text]
3. look at result

## Expected result
Inside .ui.form > div.ui.transparent.input, input element has padding: 0;

## Actual result
Inside .ui.form > div.ui.transparent.input, input element has padding: .67857143em 1em;

## Testcase
https://jsfiddle.net/j4h2prfg/9/

## Version
2.7.6

P.S. If the current behaviour is what framework is expecting, then you can close the issue

**Comment by lubber-de** [comment, marked as answer]
Yes, the behavior is wanted when inputs are inside forms, because of the consistent look when an `error` class is added to a field.
When used inside forms, any input should be wrapped by `field`.

See original discussion here https://github.com/fomantic/Fomantic-UI/issues/107

If you want no padding then use transparent inputs outside of forms.
